msteams-platform/resources/removing-tab-margins.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Removing tab margins in Microsoft Teams
+author: laujan
+description: 
+keywords: tab removing margins padding
+ms.topic: reference
+ms.author: lomeybur
+---
+
+# Removing all tab margins (2021 update)
+
+Microsoft Teams is removing the margins around all tabs. This will allow developers to build apps that look more native to Teams. This will also align with our [UI kit designs](~/tabs/design/tabs.md). Most apps already look better without the margins surrounding their experiences. However, some tabs are visually affected by this change and developers should make the necessary changes.
+
+:::image type="content" source="../../assets/images/tabs/remove-margins-tabs.png" alt-text="Tab wit and without margins" border="false":::
+
+## Timelines
+
+* 3/1 - Margins removed in [Public Developer Preview](~/resources/dev-preview/developer-preview-intro.md)
+* 5/1 - Margins will be removed in production
+
+## Guidelines
+
+Microsoft Teams apps that use tabs will be affected by this change. Developers should switch to [Public Developer Preview](~/resources/dev-preview/developer-preview-intro.md) in order to determine how their tabs are affected and make the necessary changes.
+
+Tab developers should not rely on Teams to provide margins surrounding their tabs. Developers are encouraged to add margins around their tab designs where it makes sense. App designs in production may look like there's extra padding (i.e.: margins provided by Teams, and margins provided by the tab), but this is only temporary and will go away in a few weeks, leaving only the app's provided padding.
+
+## FAQ
+
+> Is it OK for app chrome (header bar, task bar, etc) to touch the edges of our designs?
```

Comment:
Another FAQ-- What was the size of the margin that was previously applied?

Reply:
Added.
* Left and right: 20px
* Top: 16px
* Bottom: 0px